### PR TITLE
Add pre-commit configuration

### PR DESCRIPTION
**Pull request by tridelat**
Pre-commit configuration for checking commits health and to use in github actions.

### CODE REVIEW

**Comment by MaxPal** [comment]
@tridelat  Did you run the pre-commit on every files ?

**Comment by tridelat** [comment]
> @tridelat Did you run the pre-commit on every files ?

I haven't, but could and will after #53 is merged to avoid annoying conflicts

**Comment by tridelat** [comment]
@MaxPal the pre-commit was run on all files on [f1a06c8](https://github.com/Total-RD/seahowl/pull/52/commits/f1a06c8a9e8876f3875923f41f9e6823f5f2f5bb)

**Review by MaxPal — COMMENTED** [review]
DISCON.IN files are modified. Still OK for ROSCO library ?

File: src/aero/bemt.cpp [hunk 1]
```diff
@@ -128,19 +128,19 @@ chrono::ChVector2<double> seahowl::aero::get_induced_velocity(seahowl::aero::Bla
 
         // @todo fix tangential induction factor calculation (convergence)
         //// tangential induction
-        //if (fabs(cos_phi) < tol_induction) {
```
Comment (MaxPal) [inline src/aero/bemt.cpp:131]: Why keeping this section of code in comment ?

**Review by MaxPal — APPROVED** [review]
Ok for me

File: src/aero/bemt.cpp (hunk 1 above)
Comment (tridelat) [reply src/aero/bemt.cpp:131]: Here it is because it is a code we need to debug before uncommenting (see @todo), but we could remove as well

**Pull request merged**